data/roles.js

```diff
@@ -629,157 +648,163 @@ const roleData = {
-                "Once dead, may visit one person a night and roleblock them.",
+                "Once dead, visits one player each night and roleblock them.",
            ],
            graveyardParticipation: "self",
         },
         "Informant": {
             alignment: "Mafia",
             description: [
-                "Can pick a person every night.",
-                "Any system messages that person may have got are given to the informant as well"
+                "Chooses a player each night and views any system messages they recieve the following day."
             ],
          },
         "Jinx": {
             alignment: "Mafia",
             description: [
-                "Each night, curses a player with a word.",
+                "Curses a player with a forbidden word each night.",
                 "If the person speaks the word the next day, they will die."
             ],
         },
         "Clown": {
             alignment: "Mafia",
             description: [
-                "Appears as Fool to self.",
+                "Fools around at night, visiting another player.",
                 "Will kill their visit target.",
+                "Appears as Fool to self."
             ],
         },
         "Graverobber": {
             alignment: "Mafia",
             description: [
-                "Can visit a dead player every night.",
-                "Learns the role of that player and takes all items they're holding.",
+                "Visits a dead player every night.",
+                "Learns the role of that player and takes any items they were holding.",
             ],
         },
         "Medusa": {
             alignment: "Mafia",
             description: [
-                "Once per game, during the day, can turn all visitors last night to stone.",
+                "Chooses to turn last nights visitors to stone, once per game, during the day.",
+                "Players turned to stone are killed."
                 ],
         },
         "Illusionist": {
             alignment: "Mafia",
             description: [
                 "Starts with a gun.",
-                "Each night, chooses one person to frame as the shooter of any guns shot by the Illusionist."
+                "Chooses one player each night to frame as the shooter of any guns shot by the Illusionist."
             ],
         },
         "Cat Lady": {
             alignment: "Mafia",
             description: [
-                "Every day, chooses a person to send a cat to.",
-                "At night, the target can choose to let the cat in or chase it out.",
-                "If the cat is let in, the target is roleblocked.",
-                "If chased out, the Cat Lady will learn the role of the target."
+                "Chooses a player to send them a cat, each day.",
+                "The player can choose to let the cat in during the night, or chase it out.",
+                "If the cat is let in, the player is roleblocked.",
```

Comment:
Consistency. Should say "blocked from performing night actions" instead of "roleblocked."